src/lib/plugins/world.js

```diff
@@ -192,13 +192,20 @@ module.exports.player = function (player, serv, settings) {
           type: 'compound',
           name: '',
           value: {
-            MOTION_BLOCKING: { type: 'longArray', value: new Array(36).fill([0, 0]) }
+            MOTION_BLOCKING: { type: 'longArray', value: new Array(37).fill([0, 0]) }
           }
         }, // FIXME: fake heightmap
         chunkData: chunk.dump(),
         blockEntities: []
       })
-      if (registry.supportFeature('lightSentSeparately')) {
+      if (registry.supportFeature('dimensionDataIsAvailable')) {
```

Comment:
The name of the feature seems unrelated with the content of this if

Reply:
Copied directly from mineflayer code.

Reply:
https://github.com/PrismarineJS/mineflayer/blob/5c71edf48bb2f2dfa16cddb9af5baa0c4d55cf0d/lib/plugins/blocks.js#L264 definitely has the same issue